.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 *                   @zapier/epd-dev-platform-content
 /docs/_embed/       @zapier/epd-partner-sharing
 /docs/_partners/    @zapier/epd-partner-sharing
+/docs   @zapier/team-platform-support
```

Comment:
Nitpicking, but can you please match the formatting of the other `CODEOWNERS` entries?

Reply:
Fixed, VS code window was a bit narrow didn't catch this, thanks!